test/context_test.exs

```diff
@@ -240,7 +242,7 @@ defmodule ContextTest do
           ~H"""
           <Context
             put={{ ContextTest.Outer, 123 }}>
-            <slot/>
```

Comment:
This slot does not make sense as the code is compiled in a live view and live view haven't slots.

Reply:
@Malian maybe we should consider adding a `compile_surface_component` to the test API so we could generate a component instead of a LV. We only need to create a LV on-the-fly when we need to test events and side-effects. To test compilation errors in components, generating a component would be enough. I can see we currently use `Code.eval_string(...)` to do that in many places. Le me know what you think.